conwayste/src/client.rs

```diff
@@ -527,29 +530,44 @@ impl EventHandler for MainState {
                     ).unwrap(); // OK to call unwrap here because there is an else match arm (all errors handled)
                 }
 
-                if self.inputs.mouse_info.mousebutton == MouseButton::Left {
+                // TODO: move this into process_running_inputs
+                if self.inputs.mouse_info.mousebutton == MouseButton::Left && self.inputs.mouse_info.action == Some(MouseAction::Click) {
```

Comment:
Bug here. Only `MouseAction::Click`'s are allowed through, which creates dead code in the `else` branch when `self.insert_mode == None`. Result is that I cannot directly draw single cells.

I think you want to move the second-half of this conditional to line 537-539 so that only clicks will drop patterns.

Reply:
Thank you for finding this!

Reply:
Fixing this.